src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
+import Icon from 'supercons'
+
+export default function Navigation() {
+  return (
+    <NaviWrapper>
+      <NaviBar>
+        <NaviLink exact to="/">
+          <Icon glyph="home" size={24} />
+        </NaviLink>
+        <NaviLink exact to="/favorites" data-testid="favorites">
+          <Icon glyph="like" size={24} />
+        </NaviLink>
+      </NaviBar>
+    </NaviWrapper>
```

Comment:
As you are already in the `Navigation` component and this styled-components are scoped to it, you could get rid of the `Navi` prefix of all coponent names and make it more readable:
```suggestion
    <Wrapper>
      <Bar>
        <Link exact to="/">
          <Icon glyph="home" size={24} />
        </Link>
        <Link exact to="/favorites" data-testid="favorites">
          <Icon glyph="like" size={24} />
        </Link>
      </Bar>
    </Wrapper>
```

Reply:
Also: you should be able to merge `NaviBar` and `Wrapper` into one element with one styling.

Reply:
Roger!

Reply:
Roger!